new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,297 @@
+# DynamoDB Integration with Typesense
+
+Hey there! This post give you a guide to integrate Typesense custer with AWS DynamoDB by setting up a trigger with AWS Lambda.
+
+![Typesense DynamoDB Integration Chart](~@guide-images/typesense-dynamodb/dynamodb-typesense-integration-1.png)
```

Comment:
Could you use SVG logos and then export the file as an SVG as well? The current one looks quite blurry:

<img width="739" alt="Screen Shot 2021-03-10 at 8 24 55 PM" src="https://user-images.githubusercontent.com/458383/110735468-b687ad80-81de-11eb-944f-bcf7d46bdc0f.png">